omnigibson/__init__.py

```diff
@@ -97,7 +97,7 @@ def print_save_usd_warning(_):
 def create_app():
     global app
     from omni.isaac.kit import SimulationApp
-    app = SimulationApp({"headless": gm.HEADLESS})
+    app = SimulationApp({"headless": gm.HEADLESS, "multi_gpu": False})
```

Comment:
Can we add a comment here why we're forcing multi GPU to be false? (Also, should we bring this up with NV? This seems like a major regression)

Reply:
Done!